atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/encoding/TicketsEncodingStrategy.java

```diff
@@ -91,7 +90,7 @@ public long decodeValueAsCommitTimestamp(long startTimestamp, byte[] value) {
         long row = (startTimestamp / PARTITIONING_QUANTUM) * ROWS_PER_QUANTUM
                 + (startTimestamp % PARTITIONING_QUANTUM) % ROWS_PER_QUANTUM;
         byte[] rowName = ValueType.VAR_LONG.convertFromJava(row);
-        ArrayUtils.reverse(rowName);
+        EncodingUtils.reverseBits(rowName);
```

Comment:
I might be minded to use a fixed long for this (since after all if it's going in the row key, it won't show up in the index summary much) and then just use Long.reverse

Reply:
then it's just Long.reverse(PtBytes.toBytes(row)) and Long.reverse(PtBytes.fromBytes(row))

Reply:
sorry, realised that first comment was bad. The row key is a bit irrelevant for dynamic columns rows in terms of bytes, because it's only stored every few kB data; so the difference between this is well well under 1%.